pkg/kubelet/cm/cpuset/cpuset.go

```diff
@@ -319,6 +319,9 @@ func Parse(s string) (CPUSet, error) {
 			if err != nil {
 				return NewCPUSet(), err
 			}
+			if start >= end {
+				return NewCPUSet(), fmt.Errorf("invalid range \"%s\" (%d >= %d)", r, start, end)
```

Comment:
nit: `%q` adds double quotes to strings:

```suggestion
				return NewCPUSet(), fmt.Errorf("invalid range %q (%d >= %d)", r, start, end)
```